test/database_test.js

```diff
@@ -0,0 +1,38 @@
+const chakram = require('chakram');
+const expect = chakram.expect;
+
+const { testData } = require('../dao/schema');
+const db = require('../dao/database');
+const reminders = require('../dao/reminders');
+const serverManager = require('./server_manager');
+
+describe('dao', () => {
+  before(() => {
+    serverManager.reinitProfile();
```

Comment:
here you're initing once for all tests; I think we should restart properly between each test (so have it in beforeEach).